packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/ClientProviders/ClientProviderSubClientTests.cs

```diff
@@ -32,7 +36,11 @@ public void SetUp()
         [Test]
         public void ServiceClientWithSubClient()
         {
-            var client = InputFactory.Client(TestClientName);
+            var inputOperation = InputFactory.Operation("HelloAgain", parameters:
+            [
+                InputFactory.Parameter("p1", InputFactory.Array(InputPrimitiveType.String))
+            ]);
```

Comment:
it looks like that this `inputOperation` variable is not used anywhere. Is this expected?

Reply:
Right, reverted.